package.json

```diff
@@ -10,10 +10,10 @@
     "@emotion/styled": "11.14.0",
     "@hello-pangea/dnd": "18.0.1",
     "@hookform/resolvers": "^4.0.0",
-    "@mui/base": "5.0.0-beta.40",
-    "@mui/icons-material": "5.17.1",
-    "@mui/material": "5.17.1",
-    "@mui/system": "5.17.1",
+    "@mui/base": "5.0.0-beta.70",
```

Comment:
Isn't this package now deprecated? I think we're only using it for `ClickAwayListener` - which previously they pointed us to import from `@mui/base` but now the docs say to import it straight from `@mui/material`: https://v6.mui.com/material-ui/react-click-away-listener/

Reply:
Yeah it is, thanks.